pkg/cmd/update-tls-artifacts/generate-owners/generate_owners_options.go

```diff
@@ -0,0 +1,275 @@
+package generate_owners
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"strings"
+
+	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+type GenerateOwnersOptions struct {
```

Comment:
Options contain Run methods that do the work for the command once the input is set.  They *can* hold intermediate values, but usually don't and intermediate values should be private.